ouroboros-network-protocols/src/Ouroboros/Network/Protocol/LocalStateQuery/Type.hs

```diff
@@ -23,14 +23,15 @@ import           Network.TypedProtocol.Core
 
 import           Ouroboros.Network.Util.ShowProxy (ShowProxy (..))
 
+data FootprintL = SmallL | LargeL | WholeL
```

Comment:
What is it for?  What `L` stands for?

_Haddocks would be welcome._

Reply:
Gone!